universal-login-sdk/src/core/services/validations/AmountValidator.ts

```diff
@@ -1,10 +1,9 @@
 import {utils} from 'ethers';
-import {TransferDetails, SEND_TRANSACTION_GAS_LIMIT} from '@unilogin/commons';
+import {TransferDetails, SEND_TRANSACTION_GAS_LIMIT, IS_STRING_A_NUMBER} from '@unilogin/commons';
 import {Validator, TransferErrors} from './Validator';
 
 const {formatEther} = utils;
-
-const isNumber = /^[0-9]+(\.[0-9]+)?$/;
+const isNumber = IS_STRING_A_NUMBER;
```

Comment:
In my opinion, we should rename `IS_STRING_A_NUMBER` to `isStringANumberRegex` (as in case of `ensNameElementRegex`) and use is here. We shouldn't create an additional variable like `isNumber`.

Reply:
After that, we can create a function like this and move it to commons
```ts
const isStringANumber = (value?: string): boolean =>  {
    return (typeof (value) === 'string' && !!value.match(isStringANumberRegex));
  }
``` 

and use it in ValueRounder